code-challenges.rb

```diff
@@ -21,6 +37,15 @@
 album3 = 'Abbey Road'
 # Expected output: 'bby Rd'
 
+def vowel_be_gone str
+  lower_case = str.downcase
+  lower_case.delete "aeiou"
+end
+p vowel_be_gone album1
+p vowel_be_gone album2
+p vowel_be_gone album3
+
```

Comment:
I would caution you against downcasing everything since there are capital letters in the expected output.